src/api/taskRuns.js

```diff
@@ -110,9 +111,9 @@ export function createTaskRun({
     payload.spec.podTemplate = { nodeSelector };
   }
   if (params) {
-    payload.spec.params = Object.keys(params).map(name => ({
-      name,
-      value: params[name]
+    payload.spec.params = Object.keys(params).map(paramName => ({
+      paramName,
```

Comment:
Same as for the PipelineRun:

```suggestion
      name: paramName,
```